specs/windows/bitlocker_info.table

```diff
@@ -11,5 +11,6 @@ schema([
   Column("version", INTEGER, "The FVE metadata version of the drive."),
   Column("percentage_encrypted", INTEGER, "The percentage of the drive that is encrypted."),
   Column("lock_status", INTEGER, "The accessibility status of the drive from Windows."),
+  Column("protector_types", TEXT, "Indicates what types of key protectors are setup."),
```

Comment:
Nit: Should this column be an `INTEGER`?

Any documentation we can offer around possible values? Maybe just an example of a value for PIN protected?

Reply:
What @sgress454 said. 

We can add delimiters around the individual values to make querying easier, e.g. `"1", "2"` - then to determine if `TPM and PIN` is setup as a protector we could do something like `SELECT 1 FROM bitlocker_info WHERE ... AND INSTR(protector_types, '"4"')` or something like that.

I don't know if there's any guidelines around multi-value columns for this project, like:
- Should we even use them?
- If yes, what kind of separator is preferred? (", ", ",", "|", etc).
- Do the values need to ordered in a deterministic way?